chatminer/chatparsers.py

```diff
@@ -109,6 +109,8 @@ def parse_file_into_df(self):
         parsed_messages = []
         for mess in self.messages:
             parsed_mess = self._parse_message(mess)
+		    if parsed_mess['message'] == '<Media omitted>': # when the chat is exported without the media, wordcloud figure might have huge "media omitted".
+			    continue
```

Comment:
I would rather catch this case via the _stopword_ parameter in the _wordcloud_ method directly. Otherwise, we would drop the information of sent Media completely.